src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -195,6 +202,25 @@ public void onLoad(ItemGroup<? extends Item> parent, String name) throws IOExcep
         if (state == null) {
             state = new State(this);
         }
+        if (getParent() instanceof OrganizationFolder) {
+            if (new File(getRootDir(), ".jenkins-41121").isFile()) {
```

Comment:
Also, better naming, please. I don't like the idea of having even a temporary file have a bug id as its name.